test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,26 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // only one null
+        assertTrue(Utils.isAnyNull((Object)null));
+
+        // only one object
+        assertFalse(Utils.isAnyNull("abc"));
+        assertFalse(Utils.isAnyNull(1));
+
+        // more object with some null
+        assertTrue(Utils.isAnyNull("abc", null));
+        assertTrue(Utils.isAnyNull(null, 1));
+        assertTrue(Utils.isAnyNull(1, "abc", null));
+
+        // more object with no null
+        assertFalse(Utils.isAnyNull(1, "abc"));
+        assertFalse(Utils.isAnyNull(1, "abc", 2 , "ab"));
```

Comment:
Very comprehensive set of tests! However, do consider the case where the object is a list, what if there are null objects within the list?